cli/waiter/subcommands/kill.py

```diff
@@ -122,8 +122,11 @@ def register(add_parser):
     parser = add_parser('kill', help='kill services')
     parser.add_argument('token-or-service-id')
     parser.add_argument('--force', '-f', help='kill all services, never prompt', dest='force', action='store_true')
-    parser.add_argument('--timeout', '-t', help='timeout (in seconds) for kill to complete',
-                        type=check_positive, default=30)
     parser.add_argument('--service-id', '-s', help='kill by service id instead of token',
                         dest='is-service-id', action='store_true')
+    timeout_group = parser.add_mutually_exclusive_group()
+    timeout_group.add_argument('--timeout', '-t', help='timeout (in seconds) for kill to complete',
+                        type=check_positive, default=30)
+    timeout_group.add_argument('--no-wait', help="does not wait for all routers to confirm deletion", dest='no-wait',
+                        action='store_true')
```

Comment:
Can this flag be configured to set value of timeout to `0` when configured directly instead of us handling it manually?